openshift/templates/db/db-deploy.yaml

```diff
@@ -6,6 +6,37 @@ metadata:
     tags: ${NAME}${SUFFIX}
   name: ${NAME}${SUFFIX}
 objects:
+  - kind: NetworkPolicy
+    apiVersion: networking.k8s.io/v1
+    metadata:
+      name: ${NAME}
+      labels:
+        name: ${NAME}
+        app: ${APP_NAME}
+        env: ${TAG_NAME}
+    spec:
+      description: |
+        Allow the api(s) to access the db.
+      ingress:
+        - from:
+            - podSelector:
+                matchLabels:
+                  role: ${API_ROLE}
+                  # app: ${APP_NAME} # Can't fileter on APP_NAME since we use a centralized/shared wallet
```

Comment:
This is not the wallet DC

Reply:
But it is a copy/paste policy... 🤣 (fixed)